spec/controllers/coronavirus_form/essential_supplies_controller_spec.rb

```diff
@@ -57,8 +57,17 @@
       expect(response).to render_template(current_template)
     end
 
-    it "redirects to check your answers if check your answers previously seen" do
+    it "redirects to to next question if check your answers previously seen and later food questions not answered" do
       session[:check_answers_seen] = true
+      post :submit, params: { essential_supplies: I18n.t("coronavirus_form.questions.essential_supplies.options.option_no.label") }
+
+      expect(response).to redirect_to(dietary_requirements_path)
+    end
+
+    it "redirects to check your answers if check your answers previously seen and later food questions answered" do
```

Comment:
Can you add a test for, `check_answers_seen` and later food questions not answered, and user answers "yes" to `essential_supplies`, as I think the behaviour needs to be different depending on the answer to that question.

Reply:
Added this.  Also picks up the missing logic you have raised in the controller.